src/AppRoutes.js

```diff
@@ -7,20 +7,17 @@ import Homepage from './containers/Home/Home';
 import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import ProtectedRoute from './containers/ProtectedRoute';
-
-const Login = () => (
-  <div><h1>Login page</h1></div>
-);
+import SignIn from './containers/SignIn/SignIn'; // eslint-disable-line
```

Comment:
do not disable eslint. If you have named and default exports with the same name, a quick fix will be making a default import with with a different name e.g Signin instead of SignIn if you want to use default